libs/api/src/lib/graphql/query.private.ts

```diff
@@ -748,6 +752,27 @@ export const GraphQLQuery = new GraphQLObjectType<undefined, Context>({
         id: {type: GraphQLID}
       },
       resolve: (root, {id}, {prisma: {event}}) => getEvent(id, event)
+    },
+
+    // Stats
+    newSubscribersPastYear: {
```

Comment:
I like this idea. :+1:

Reply:
I would not suggest to do this because this would be how REST APIs would be designed but not GraphQL.
GraphQL's intention was so that the client decides what it wants and it does not have to be integrated into the server.

If we have multiple endpoints one can do:

```graphql
query Stats($take: Int, $skip: Int) {
  newSubscribersPastYear() {
     ....
  }

  newArticlesPastYear() {
    ....
  }

  newPagesPastYear() {
    ....
  }
}
```

Which would give you everything as 1 call.

Reply:
actually, I can agree with that.